services/frontend-v3/src/components/education/EducationView.jsx

```diff
@@ -17,6 +17,19 @@ const EducationView = ({ educationInfo }) => {
     },
   };
 
+  const getUrl = (item) => {
+    if (item.attachmentLinks && item.attachmentLinks.length > 0)
+      return item.attachmentLinks.map((i) => (
+        <Tag color="rgb(114, 114, 114)" key={i.id}>
+          <LinkOutlined />
+          <a target="_blank" rel="noreferrer" href={i.url}>
+            {i.name}
+          </a>
+        </Tag>
+      ));
+    return <></>;
```

Comment:
Here you can just return null or undefined (to keep consistency with the rest of the code)